acoustid/future/fpindex/client.py

```
@@ -0,0 +1,396 @@
+                        f"Unexpected status {status}: {body}"
+                    )
+
+                if expected_response is None:
+                    return status, None
+
+                content = await response.read()
+                return status, msgspec.msgpack.decode(content, type=expected_response)
+
+        except aiohttp.ClientError as e:
+            logger.exception("Error making request")
+            raise FingerprintIndexClientError(f"Error making request: {e}") from e
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)